.github/workflows/ci.yml

```diff
@@ -20,13 +20,13 @@ jobs:
           node-version: '12'
       - run: npm install -g npm
       - run: npm install
-      - run: npm run format
```

Comment:
I added this change here so I don't have to deal with `build` after `format` stopping my CI midway through.